index.html

```diff
@@ -122,6 +122,34 @@ <h2>A project for first-time contributions</h2>
 
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Natasha Ringane</p>
+          <p class="contact">
+            <i class="fab fa-instagram"></i>
+            <a href="https://www.instagram.com/__snow.flakee/" target="_blank">@__snow.flakee</a>
+            <i class="fab fa-github"></i>
+            <a href="https://www.github.com/natasha-snow" target="_blank">natasha-snow</a>
+          </p>
+          <p class="about">I am an aspiring front-end developer, hoping to be a full-stack
```

Comment:
Small feedback: typically, you wouldn't see text split into two lines like this. :) We'd generally keep it all on one line and leave it to our code editor of choice to handle line wrapping for us. 

It doesn't really do anything _bad_ to the code -- HTML handles whitespace the same whether it's a space, a tab, or a new line break like this -- but I wanted to give you some feedback to help you increase your knowledge. 🧠💪😎